### DISCUSSION TITLE
Hide volume text but still show bar

### DISCUSSION THREAD

**Discussion by advice2020** [mpv-player/mpv#15073] [General]
I see that you can turn off the osd-bar for volume but the text still shows.
I would like to do the opposite, hide the text and only show the bar.
Does anyone know if this is possible?

Thank You

**Comment by guidocella** [comment]
Bind `9 osd-msg add volume -2` and `0 osd-msg add volume 2`.

**Comment by advice2020** [comment]
@guidocella 
Thank You
I have seen this mentioned somewhere else as well but I think this is for the opposite of what I am looking for?
With these two keybinds the volume text shows and the volume bar does not.
I am looking for the volume bar to show and to hide the volume text.

Can you or someone verify if this suggestion was made by mistake, or if I am not doing something correctly?

I do not have the following setting in my mpv.conf set so it is using the default of “yes”
`--osd-bar=<yes|no>`

Should have no relation but I also have this setting in my mpv.conf which is for the duration OSD bar to not show
`osd-on-seek=no`

Thank You

**Reply by guidocella** [reply]
Yeah you need `osd-bar add volume 5`.

**Comment by advice2020** [comment]
@guidocella 
Ahhh that is it, it is a Input Command Prefix
Thank you very much for this :)
